specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -0,0 +1,6389 @@
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DesktopVirtualization/hostPools": {
+      "get": {
+        "tags": [
+          "HostPool"
+        ],
+        "operationId": "HostPools_ListByResourceGroup",
+        "description": "List hostPools.",
+        "x-ms-examples": {
+          "HostPool_ListByResourceGroup": {
+            "$ref": "./examples/HostPool_ListByResourceGroup.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved host pool in resource group.",
+            "schema": {
+              "$ref": "#/definitions/HostPoolList"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DesktopVirtualization/hostPools": {
+      "get": {
+        "tags": [
+          "HostPool"
+        ],
+        "operationId": "HostPools_List",
+        "description": "List hostPools in subscription.",
+        "x-ms-examples": {
+          "HostPool_List": {
+            "$ref": "./examples/HostPool_List.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved host pool in subscription.",
+            "schema": {
+              "$ref": "#/definitions/HostPoolList"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DesktopVirtualization/hostPools/{hostPoolName}/update": {
```

Comment:
With the proposal of merging initiate and validate - is the flag: ValidateOnly going to be passed in as query string param? @datr93

Reply:
Its passed in with the post:

https://stackoverflow.com/questions/5876809/do-http-post-methods-send-data-as-a-querystring/5876931